docs/04-operation-guides/operations/evnt-03-nats-server-system-events.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Acquiring NATS server system account credentials
+---
+## Context
+
+Accessing certain resources in NATS requires [`system_account` privileges](https://docs.nats.io/running-a-nats-service/configuration/sys_accounts). Kyma automatically generates a `system account` user using a Secret dubbed `eventing-nats-secret` in the `kyma-system` Namespace.
+
+## Prerequisites
+
+Kyma version `2.7` or later.
```

Comment:
wrong version. 2.10 it is...
or just omit the version here. it will show up as soon as it is available